cloud/bin/cloud.ts

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+import { App } from 'aws-cdk-lib';
+import 'source-map-support/register';
+
+import {
+	appName,
+	resourceDescription,
+	stackName,
+	stageName,
+	ApiStack,
+} from '../lib';
+
+const app = new App();
+const tags = {
```

Comment:
Worth adding an owner tag? I remember being asked for that on grad project, but don't know what the company policy is

Reply:
@pmarsh-scottlogic FYI We need to request a new, project-specific AWS account instead of using Dev Edinburgh. Then I think tags will be less important. I've requested an account today.